packages/itwin/ec3-widget/src/ec3-widget-react.ts

```diff
@@ -14,3 +14,4 @@ export { Templates, TemplateProps } from "./components/Templates";
 export { TemplateMenu, TemplateMenuProps } from "./components/TemplateMenu";
 export { Configuration } from "./components/EC3/Template";
 export { EC3Token } from "./components/EC3/EC3Token";
+export { EC3WidgetUI } from "./EC3WidgetUI";
```

Comment:
Looking at the other packages in this repository, it looks like this should be named `EC3Widget`. I also see that there already exists `EC3Widget.tsx`, however it contains a class `EC3Provider`, therefore it should be renamed to `EC3Provider.tsx`.

`EC3Widget.tsx` -> `EC3Provider.tsx`
`EC3WidgetUI.ts` -> `EC3Widget.ts`

Reply:
done